xcode/Subconscious/Shared/Components/Common/Profile/ProfilePic.swift

```diff
@@ -7,21 +7,58 @@
 
 import SwiftUI
 
-struct ProfilePic: View {
+enum ProfilePicVariant: Equatable, Codable, Hashable {
+    case none
+    case url(URL)
+    case image(String)
+}
+
+struct ProfilePicImage: View {
```

Comment:
Nit: alternatively, should this be a view modifier? That way we could pass ordinary `Image` views.

Reply:
I don't feel strongly either way.